.github/workflows/pull-request.yml

```diff
@@ -2,11 +2,13 @@ name: Pull request created
 
 on:
   pull_request:
-    branches: [ master ]
     types: [edited, synchronize]
+    paths-ignore:
+      - 'develop/**'
 
 jobs:
   test:
+    if: github.event.pull_request.head.ref != 'develop' && contains( github.event.pull_request.labels.*.name, 'deploy-pr')
```

Comment:
Why do we want to limit it? Does building/testing take too much time?

Reply:
Wikigdrive starts watching each shared drive for changes. It seems that the more watchers we use the slower they work. Each additional instance is another set of connections to the API.